happi/containers.py

```diff
@@ -297,3 +297,43 @@ class Motor(Device):
     device_class.default = 'pcdsdevices.device_types.Motor'
     system = copy(Device.system)
     system.default = 'motion'
+
+
+class Camera(Device):
+    """
+    A Generic EpicsCamera
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Camera'
```

Comment:
What is the difference between `Camera` and `AreaDetector`? If the `Camera` objects are in the DAQ, let's ignore them for now.

Reply:
Removed reading cameras from questionnaire due to them not having PV's, will add back later once questionnaire is patched